src/containers/Auth/ConfirmEmailPage.js

```diff
@@ -29,7 +29,7 @@ function ConfirmEmailPage(props) {
     confirmEmail(query).catch(() => {
       history.push('/login');
     });
-  }, [confirmEmail, history]);
+  }, [confirmEmail, history, query]);
```

Comment:
Remove this for now, as I remember it was making multiple requests if we add this

Reply:
fixed